contracts/examples/ping-pong-egld/wasm/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ping-pong-wasm"
+version = "0.0.1"
+authors = [ "you",]
+edition = "2018"
```

Comment:
we now started adding
publish = false
to all sample contracts

Reply:
done